parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -615,14 +656,31 @@ public boolean equals(Object obj) {
     public int hashCode() {
       return Objects.hash(isAdjustedToUTC, unit);
     }
+
+    @Override
+    PrimitiveStringifier valueStringifier(PrimitiveType primitiveType) {
+      switch (unit) {
+        case MICROS:
+          return PrimitiveStringifier.TIMESTAMP_MICROS_STRINGIFIER;
```

Comment:
Could you please also modifiy the TIMESTAMP_*_STRINGIFIER-s to append " UTC" to its output if the timestamps are UTC-normalized?

Reply:
I think it should be part rather of #513 no?

Reply:
SGTM